src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx

```diff
@@ -342,11 +346,17 @@ const UniqueTokenExpandedStateHeader = ({
 
   const onPressAndroidAsset = useCallback(() => {
     const androidContractActions = [
-      'View On Web',
-      // @ts-expect-error network could be undefined?
-      `View on ${startCase(ethereumUtils.getBlockExplorer(asset?.network))}`,
-      ...(isPhotoDownloadAvailable ? (['Save to Photos'] as const) : []),
-      'Copy Token ID',
+      lang.t('expanded_state.unique_expanded.view_on_web'),
+      lang.t('view_on_block_explorer', {
```

Comment:
is this meant to be `expanded_state.unique_expanded.view_on_block_explorer`

Reply:
@jinchung Yes, fixing now!